manifests/rootsync-crd.yaml

```diff
@@ -1202,7 +1202,7 @@ spec:
                   values:
                     description: values to use instead of default values that accompany
                       the chart. Format values the same as default values.yaml. These
-                      values will take precedence if used in conjunction with valuesFileSources.
+                      values will take precedence if used in conjunction with valuesFileRefs.
```

Comment:
"if used in conjunction with valuesFileRefs" -> "if valuesFileRefs is also specified"